docs/blog/introducing-modern-web/index.md

```diff
@@ -106,15 +106,15 @@ Not only do we ship [rollup-plugin-workbox](https://www.npmjs.com/search?q=rollu
 
 In the past years, we've used different frameworks and tools to build our websites. Recently we've become big fans of [11ty](https://www.11ty.dev/). It's simple to use, works with markdown, and generates _just_ plain HTML. There is no runtime javascript involved with displaying page content, making it super fast.
 
-To add interactivity to our page we started using web components, and applied rollup with our [HTLM](https://www.npmjs.com/package/@open-wc/rollup-plugin-html) and [workbox](https://www.npmjs.com/search?q=rollup-plugin-workbox) plugins to the HTML pages generated by 11ty.
+To add interactivity to our page we started using web components, and applied rollup with our [HTML](https://www.npmjs.com/package/@open-wc/rollup-plugin-html) and [workbox](https://www.npmjs.com/search?q=rollup-plugin-workbox) plugin.
```

Comment:
```suggestion
To add interactivity to our page we started using web components, and applied rollup with our [HTML](https://www.npmjs.com/package/@open-wc/rollup-plugin-html) and [workbox](https://www.npmjs.com/search?q=rollup-plugin-workbox) plugins.
```

Reply:
`and applied rollup with our html and workbox plugins.`  sounds strange to me - but no clue 😅

Reply:
Well it's plural :) The original sentence was `and applied rollup with our HTML and workbox plugins to the HTML pages generated by 11ty`. Not great either, so maybe we should rephrase it entirely 🤔

Reply:
`and added rollup using our HTML and workbox plugins`?

Reply:
change it to 

To add interactivity to our page we started using web components and applied rollup with our plugins for [HTML](https://www.npmjs.com/package/@open-wc/rollup-plugin-html) and [workbox](https://www.npmjs.com/search?q=rollup-plugin-workbox).

sounds good?

Reply:
Yeah 👍